crates/ruma-signatures/src/functions.rs

```diff
@@ -983,12 +984,13 @@ mod tests {
 
         assert!(verification_result.is_err());
         let error_msg = verification_result.err().unwrap().message;
-        assert!(error_msg.contains("signature verification failed"));
+        assert!(error_msg.contains("Could not verify signature"));
```

Comment:
This error message happens at "the last stage", when the library cannot verify the signature with the public key its given, the error generated with parsing the public key will more or less be about it a. being a valid length (64), or b. being a valid key.

This test swaps the keys, so its still valid, and has a valid length, so the error happens at the actual verification spot.